src/launcher.rs

```diff
@@ -42,7 +42,12 @@ use std::env;
 /// ```
 ///
 pub fn start<W: Send + 'static>(world: W, register_fns: &[&Fn(&mut CucumberRegistrar<W>)]) {
-  start_with_addr("0.0.0.0:7878", world, register_fns)
+  let addr = if cfg!(target_os = "windows") {
+    "127.0.0.1:7878"
+  } else {
+    "0.0.0.0:7878"
```

Comment:
So the subconscious intent of this binding (on linux anyway) was that it lets the server be visible externally. That's not a common use case as far as I can tell. In order to keep the behavior the same, we might want to just bind to "127.0.0.1:7878" on all OSes.

I know this doesn't make a material difference to you since you're on windows, but what do you think about removing the special logic and just using localhost directly in all cases?